datasets/titanic/titanic_by_openml_converter.json

```diff
@@ -0,0 +1,124 @@
+      "encodingFormat": "application/vnd.apache.parquet",
+      "name": "Titanic (parquet)"
+    },
+    {
+      "@type": "FileObject",
+      "contentUrl": "https://api.openml.org/data/v1/download/16826755/Titanic.arff",
+      "encodingFormat": "text/plain",
+      "name": "Titanic (arff)"
+    }
+  ],
+  "isAccessibleForFree": true,
+  "keywords": "text_data",
+  "name": "Titanic",
+  "recordSet": [
+    {
+      "@type": "ml:RecordSet",
+      "field": [
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "pclass",
+          "source": "#{Titanic (parquet)/pclass}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Boolean",
+          "name": "survived",
+          "source": "#{Titanic (parquet)/survived}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "name",
+          "source": "#{Titanic (parquet)/name}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "sex",
+          "source": "#{Titanic (parquet)/sex}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "age",
+          "source": "#{Titanic (parquet)/age}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "sibsp",
+          "source": "#{Titanic (parquet)/sibsp}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "parch",
+          "source": "#{Titanic (parquet)/parch}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "ticket",
+          "source": "#{Titanic (parquet)/ticket}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "fare",
+          "source": "#{Titanic (parquet)/fare}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "cabin",
+          "source": "#{Titanic (parquet)/cabin}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "embarked",
+          "source": "#{Titanic (parquet)/embarked}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "boat",
+          "source": "#{Titanic (parquet)/boat}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Number",
+          "name": "body",
+          "source": "#{Titanic (parquet)/body}"
+        },
+        {
+          "@type": "ml:Field",
+          "dataType": "Text",
+          "name": "home.dest",
+          "source": "#{Titanic (parquet)/home.dest}"
+        }
+      ],
+      "name": "Titanic",
+      "source": "#{Titanic (parquet)}"
+    }
+  ],
+  "url": "https://www.openml.org/api/v1/json/data/40945",
```

Comment:
This is a JSON containing roughly the same data as specified in other fields, like description, right?
Should we have some guidelines on what this URL should prefer pointing at? Machine readable vs human friendly URL? The later with schema.org embedded machine readable info?

@benjelloun

Reply:
Good point! I changed the URL to a human friendly page that describes the dataset. This is also more in line with https://developers.google.com/search/docs/appearance/structured-data/dataset, where they describe the `url` as "Location of a page describing the dataset.".

Fixed in https://github.com/mlcommons/datasets_format/pull/3/commits/1a81c16e3a9bbe96471da88ccd91f6bc24e31dc9